### DISCUSSION TITLE
pan-os-php framework maintenance - continuity

### DISCUSSION THREAD

**Discussion by swaschkut** [PaloAltoNetworks/pan-os-php#787] [General]
Dear community,

as I left Palo Alto Networks end of 2023, there are and will be no new features commit to this GitHub repository from my side.

As pan-os-php was always done via an OpenSource License ISC -
starting by January 2024 all bug fixes or new features will be done through:
**https://github.com/swaschkut/pan-os-php**

For Docker Container environement no changes are needed, as this was already done in the past through a personal Docker Hub Account:
**https://hub.docker.com/search?q=swaschkut**

Another new interaction is now available via SLACK:
**https://join.slack.com/t/pan-os-php/shared_invite/zt-25jpjknqa-S7vm3keLSDT5kXAFSOCoGg**

or the old one via Reedit:
https://www.reddit.com/r/pan_os_php/

**Comment by bethatasitmay** [comment]
Assuming this is a desired transition, congratulations!  If not, so sorry!

Sad to see you go either way - I know you say you'll continue development, but things like that eventually fade away due to time commitments - hard to justify it when it's not your job.

Will you be enabling discussions on your github or are you only keeping it on Slack/Reddit?  I hate Slack because it's pay-to-play - want to see posts older than 90 days?  Pay for it.  Whereas github discussions or better yet Reddit are free (well I don't know if you have to pay anything).

But, regardless of the above, thank you so much for what you have done and what you may continue to do.

**Reply by swaschkut** [reply]
great suggestion.
Discussion and Issue are now enabled on;
https://github.com/swaschkut/pan-os-php

As I myself are using pan-os-php in my new job position,
this is of course the approach why I am continuing this framework.

I still need to figure out how to clone issue and wiki

**Reply by bethatasitmay** [reply]
Excellent news! :-)
